### ISSUE TITLE
Debian 12, installation problem

### ISSUE THREAD

**Issue by im-vmn** [Tribler/tribler#8314]
It's not a bug, so I publish it as a question. Steps:

1. Download the .deb file from tribler.org
2. Run `sudo dpkg -i tribler_8.0.5_x64.deb`

Result:

```
Preparing to unpack tribler_8.0.5_x64.deb ...
Unpacking tribler (8.0.5) ...
dpkg: dependency problems prevent configuration of tribler:
 tribler depends on gir1.2-appindicator3-0.1; however:
  Package gir1.2-appindicator3-0.1 is not installed.

dpkg: error processing package tribler (--install):
 dependency problems - leaving unconfigured
Processing triggers for desktop-file-utils (0.26-1) ...
Processing triggers for mailcap (3.70+nmu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Errors were encountered while processing:
 tribler
```

I always install tribler like this and before this release everything was fine. Tribler was actually installed and I can use it, but this error confuses me, moreover when I try to run apt upgrade I run into this:

```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 tribler : Depends: gir1.2-appindicator3-0.1 but it is not installable
E: Broken packages
```

**Comment by qstokkink** [comment]
We recently fixed this in #8277 but it is not part of a release (yet). We'll try to get a new release out with the fix A.S.A.P. (hopefully even today).

[If you really want it, the build in [https://github.com/Tribler/tribler/issues/8277#issuecomment-2506556498](https://github.com/Tribler/tribler/issues/8277#issuecomment-2506556498) should work for you - but this is not an official release]

**Comment by im-vmn** [comment]
Thank you
